alonet/panoptic/coco_data_modules.py

```diff
@@ -0,0 +1,59 @@
+"""`Pytorch Lightning Data Module <https://pytorch-lightning.readthedocs.io/en/latest/extensions/datamodules.html>`_
+connector between dataset and model. Load train/val/test sets and make the preprocessing required for use by the
+architecture.
+"""
+
+from typing import Optional
+
+import alodataset
+import alonet
+
+
+class CocoPanoptic2Detr(alonet.detr.CocoDetection2Detr):
```

Comment:
I think it is time to slightly organised the detr data modules into one folder:

data_modules /
    data2detr.py : include global transformations and common parameters
    coco2detr.py : include coco dataset specific methods
    cocopanoptic2detr.py include coco panoptic dataset specific methods

Reply:
Good idea. Done and tested